web/src/main/java/org/akaza/openclinica/controller/openrosa/OpenRosaSubmissionController.java

```diff
@@ -193,7 +194,9 @@ private boolean mayProceed(String studyOid, StudySubjectBean ssBean) throws Exce
         return accessPermission;
     }
 
-    public static String getAttachedFilePath(String studyOid) {
+    public static String getAttachedFilePath(String inputStudyOid) {
+    	// Using a standard library to validate/Sanitize user inputs which will be used in path expression to prevent from path traversal
+    	String studyOid =  FilenameUtils.getName(inputStudyOid);
```

Comment:
same comment as above

Reply:
changed--check for canonical path  and only allow if the canonical path starts with the base dir path.